app/components/HowCanIHelp/howcanihelp.css

```diff
@@ -3,3 +3,7 @@
   padding-top: var(--spacing-104);
   margin-top: var(--spacing-104);
 }
+
+.link-card .image+div {
+  max-width: calc(100% - 105px);
```

Comment:
I don't like this kind of hackery, but it gets the job done:
![image](https://github.com/user-attachments/assets/59b09567-add3-4f88-841a-82037fe76ac2)